web-starter/index.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var generators = require('yeoman-generator');
+var ejs = require('ejs');
```

Comment:
Why are you using EJS directly rather than using the Yeoman built-in templating facilities?

Reply:
The gitignore combining function in the app generator doesn't run the gitignore items through EJS rendering